### DISCUSSION TITLE
Set column descriptions via name lookup

### DISCUSSION THREAD

**Discussion by deanevs** [dbeaver/dbeaver#10785] [General]
I work with a database that has hundreds of meaningless names, many repeated across tables.  I would like to set the descriptions of the columns from this lookup so at least I can see what it is with mouse over column title.

Thanks for any help.

Dean

**Comment by aisbergde** [comment]
I would like to detail that:

In Access, I can use lookup tables for fields, so instead of displaying the FK, you can display a field from the linked table (for example, a name instead of an ID).

In dbeaver I am already positively surprised that you can show the linked table on the right side if there is a real FK-PK relationship. Maybe this even works with virtual keys (I haven't tested yet).

If you can already show the value on the right side in an extra window, then maybe you could even go one step further and allow to show a lookup value directly inline in the data grid?

**Comment by aisbergde** [comment]
I mean something like this, but directly integrated into the grid:
![image](https://user-images.githubusercontent.com/9020187/108418257-474b1900-7231-11eb-8698-28086308c061.png)
